src/store/actions/createComment.js

```diff
@@ -27,7 +29,14 @@ export default function createComment ({
       postId,
       text,
       attachments,
-      analytics: AnalyticsEvents.COMMENT_CREATED
+      analytics: {
+        eventName: AnalyticsEvents.COMMENT_CREATED,
+        commentLength: TextHelpers.textLengthHTML(text),
+        groupId: post.groups.map(g => g.id),
```

Comment:
Just curious why you called it `groupId` vs `groupIds` and always require an array (which may be the case).

Reply:
Thats how Mixpanel wanted it, they wanted a group Key and suggested it be singular, but then you can pass in an array for multiple values